bangazonapi/views/cart.py

```diff
@@ -112,19 +112,19 @@ def list(self, request):
             open_order = Order.objects.get(
                 customer=current_user, payment_type=None)
 
-            products_on_order = Product.objects.filter(
-                lineitems__order=open_order)
+            # products_on_order = Product.objects.filter(
```

Comment:
This `cart` view is kind of a red herring... the `cart` we've been working with so far is actually a custom action on the `profileview` class. The changes you made to the `profile.py` file were sufficient to resolve the `lineitems` v `line_items` issue!